apps/accounts/automations/change_secret/manager.py

```diff
@@ -58,7 +57,6 @@ def get_or_create_record(self, asset, account, name):
         self.name_recorder_mapper[name] = recorder
         return recorder
 
-    @bulk_create_decorator(ChangeSecretRecord)
     def create_record(self, asset, account, new_secret):
         recorder = ChangeSecretRecord(
             asset=asset, account=account, execution=self.execution,
```

Comment:
There are no significant differences between the provided code blocks at this time. However, you might consider refactoring some of the existing code to make it more readable and maintainable if your team uses that programming style convention.